client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js

```diff
@@ -230,6 +230,60 @@ export default class {
       }).$promise;
   }
 
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:tucVoipServiceAlias#fetchContactCenterSolutionNumberQueueStatistics
+   *  @methodOf managerApp.service:tucVoipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch statistics of a contact center solution queue.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *  @param  {String}      queueId               Id of queue to get stats
+   *
+   */
+  fetchContactCenterSolutionNumberQueueStatistics({ billingAccount, serviceName }, queueId) {
+    return this.OvhApiTelephony.EasyHunting().Hunting().Queue().v6()
+      .getLiveStatistics({
+        billingAccount,
+        serviceName,
+        queueId,
+      }).$promise;
+  }
+
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:tucVoipServiceAlias#fetchContactCenterSolutionNumberQueueCalls
+   *  @methodOf managerApp.service:tucVoipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch calls of a contact center solution queue.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *  @param  {String}      queueId               Id of queue to get stats
+   *
+   */
+  fetchContactCenterSolutionNumberQueueCalls({ billingAccount, serviceName }, queueId) {
+    return this.OvhApiTelephony.EasyHunting().Hunting().Queue().LiveCalls()
+      .v6()
+      .query({
+        billingAccount,
+        serviceName,
+        queueId,
+      }).$promise
+      .then(callsIds => this.$q
+        .all(callsIds.reverse().map(
```

Comment:
Please check that `callsIds` is not a string

Reply:
Why ? query request are defined by `isArray: true` (see: https://github.com/ovh-ux/ovh-api-services/blob/master/src/telephony/easyHunting/hunting/queue/liveCalls/telephony-easyHunting-hunting-queue-liveCalls.v6.service.js#L12)

Reply:
`string` is an `array`: `['e', 'r', 'r', 'o', 'r']`

We've had issues with the API sending us error in strings and I'm not sure `isArray: true` prevents this issue (deserves a try though)